keeweb/lib/Migration/AddMimetypeToFilecache.php

```diff
@@ -2,29 +2,26 @@
 
 namespace OCA\Keeweb\Migration;
 
-use OCP\Migration\IRepairStep;
-use OCP\IDBConnection;
+use OCP\Files\IMimeTypeLoader;
 use OCP\Migration\IOutput;
+use OCP\Migration\IRepairStep;
 
 class AddMimetypeToFilecache implements IRepairStep {
-  public function __construct(IDBConnection $connection) {}
-
-  public function getName() {
-    return "Add custom mimetype to filecache";
-  }
 
-  public function run(IOutput $output) {
-    $mimeTypeDetector = \OC::$server->getMimeTypeDetector();
-    $mimeTypeLoader = \OC::$server->getMimeTypeLoader();
+    private $mimeTypeLoader;
 
-    // Register custom mimetype
-    $mimeTypeDetector->getAllMappings();
-    $mimeTypeDetector->registerType('kdbx', 'x-application/kdbx', 'x-application/kdbx');
+    public function __construct(IMimeTypeLoader $mimeTypeLoader) {
+        $this->mimeTypeLoader = $mimeTypeLoader;
+    }
 
-    // And update the filecache for it.
-    $mimetypeId = $mimeTypeLoader->getId('x-application/kdbx');
```

Comment:
https://github.com/jhass/nextcloud-keeweb/blob/380e78ffbc21d3edaf8b10dc0132ccc708df46a3/keeweb/js/viewer.js#L21

Do not work because they mimetype is created as `x-application/kdbx` over here.